console/console-init/ui/src/modules/address/components/AddressConfiguration/AddressConfiguration.tsx

```diff
@@ -144,6 +156,22 @@ const AddressConfiguration: React.FunctionComponent<IAddressConfigurationProps>
                 />
               </FormGroup>
             )}
+            {(type?.toLowerCase() === "subscription" ||
+              type?.toLowerCase() === "queue") && (
+              <>
+                <ExpiryQueue
+                  expiryQueue={expiryQueue}
+                  onExpiryQueueSelect={onExpiryQueueSelect}
+                  deadletterOptions={deadletterOptions}
+                />
+
+                <Deadletter
```

Comment:
The component names are inconsistent. Either should have Queue as the suffix or both should exclude it.